_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -595,3 +595,59 @@ def get_geo_values(self, geo_type):
             crosswalk = pd.read_csv(stream, dtype=str)
             self.geo_lists[geo_type] = set(crosswalk[geo_type])
             return self.geo_lists[geo_type]
+
+    def get_geos_within(self, container_geocode, child_geocode_type,container_geocode_type):
+        """
+        Return all contained/child regions.
+
+        Given a container geocode (e.g. "ca" for California, a state)
+        and an enclosed child geocode type (e.g. "county").
+        return a set of container geocode value of the specified type that
+        lie within the specified geo code (e.g. all counties within California)
+        Support these 3 types:
+            - all states within a nation
+            - all counties within a state
+            - all states within an hhs region
+
+        Parameters
+        ----------
+        container_geocode: str of identity of nation/state/hhs
+            "fips" for return container_geocode of state
+            "state_id" for return container_geocode of nation and hhs
+        child_geocode_type: str
+            One of "state","county"
+        container_geocode_type: str
+            One of "state","nation","hhs"
+
+        Returns
+        -------
+        Set of geo ids of the specified type that lie within the specified container geocode,
+        all in string format.
+        """
+        if child_geocode_type not in ("county","state"):
+            raise ValueError("child_geocode_type must be one of state, nation and hhs")
+        get_geo_values=set()
+        if child_geocode_type=="state":
+            if container_geocode_type=="nation" and container_geocode=="us":
+                crosswalk = self._load_crosswalk_from_file("state", "state")
+                get_geo_values=set(crosswalk["state_id"])
```

Comment:
Ah so `get_geo_values` sounds like a function name (verb_noun), I would suggest renaming it to just `geo_values` (noun). That's the typical style I try to keep consistent, so that functions and variables can be easily distinguished.

Reply:
Gotcha!